src/modules/ROOT/pages/changelog/202303/05.json

```diff
@@ -87,47 +76,47 @@
     },
     {
       "what": [
-        "drill-down option in metric page block not working if the metric label wasn't provided."
+        "drill-down option in the metric page block not working if the metric label wasn't provided."
       ],
       "refs": [
         "https://github.com/cortezaproject/corteza/commit/95ae09e80c35ec24f21379a246f9c3c8e2654a5e"
       ]
     },
     {
       "what": [
-        "removing of unnecessary resources in device memory after switching from one page to another in {APP_NAME_COMPOSE}."
+        "memory leak issue for {APP_NAME_COMPOSE} when switching between different pages.."
       ],
       "refs": [
         "https://github.com/cortezaproject/corteza/commit/95cf245daafc521fc25e0f3bb20b5e2e9ed1b94a"
       ]
     },
     {
       "what": [
-        "not being able to save a page after any page block is removed from a tabs page block."
+        "not being able to save a page after removing a page block from a tabs page block."
       ],
       "refs": [
         "https://github.com/cortezaproject/corteza/commit/5addb87b3b339493e2c9788aca0b42a879264663"
       ]
     },
     {
       "what": [
-        "visibility of conditional field in a record."
+        "conditional field being displayed even if its condition says the opposite."
```

Comment:
Better -- it elaborates.
My only note here is that I'm not the biggest fan of writing something says the opposite.
Might be just me so ignore if it's me.

I'd do "conditional fields (notice the plural here; do use the plural in yours regardless) being displayed even though their condition evaluates to `false`"

Reply:
Extra not here, this issue only happened when viewing records, and not when editing. Might be worth noting